helm/values.yaml

```diff
@@ -83,11 +86,19 @@ resources: {}
   #   memory: 128Mi
 
 autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 100
-  targetCPUUtilizationPercentage: 80
-  # targetMemoryUtilizationPercentage: 80
+  shard:
+    enabled: true
+    minReplicas: 3
+    maxReplicas: 6
+    targetCPUUtilizationPercentage: 80
+    targetMemoryUtilizationPercentage: 80
+  # telemetry-core is not scalable atm.
+  core:
+    enabled: false
+    minReplicas: 3
+    maxReplicas: 6
+    targetCPUUtilizationPercentage: 80
+    targetMemoryUtilizationPercentage: 80
```

Comment:
I'm assuming that these lines don't mean anything since `enabled: false` above? Can they be removed/set to min/max or 1?

Reply:
they don't do anything, we don't use this variables in any helm template files, so setting `enabled:` to `true` or `false` would affect the deployment at all. they are just there for later use when substrate-core becomes scalable.